cicd/2-cicd/cicd.template.yml

```diff
@@ -395,6 +395,28 @@ Resources:
                   Capabilities: CAPABILITY_AUTO_EXPAND
                   RoleArn: !Sub arn:aws:iam::${AWS::AccountId}:role/admin/CloudFormationService
           - !Ref AWS::NoValue
+
+        - !If
+          - DeployForProduction
+          - Name: Smoke_Test
+            Actions:
+              - Name: smoke-test
+                ActionTypeId:
+                  Category: Build
+                  Owner: AWS
+                  Provider: CodeBuild
+                  Version: 1
+                InputArtifacts:
+                  - Name: sourceCode
+                Configuration:
+                  ProjectName: !Ref IntegrationTestBuildProject
```

Comment:
Just want to make sure I understand, how are these different from integration tests?

Reply:
The only difference is the environment variables for the domain/subdomain.